src/Paramore.Brighter.Extensions.Hosting/TimedOutboxSweeper.cs

```diff
@@ -115,7 +115,7 @@ private void Sweep(object state)
                     IAmACommandProcessor commandProcessor = scope.ServiceProvider.GetService<IAmACommandProcessor>();
 
                     var outBoxSweeper = new OutboxSweeper(
-                        millisecondsSinceSent: _options.MinimumMessageAge,
+                        timeSinceSent: TimeSpan.FromMilliseconds(_options.MinimumMessageAge),
```

Comment:
Change the options MinimumMessageAge to be a TimeSpan

Reply:
Agreed, Fixed